SPECS/kernel/config_aarch64

```diff
@@ -153,8 +160,8 @@ CONFIG_IPC_NS=y
 CONFIG_USER_NS=y
 CONFIG_PID_NS=y
 CONFIG_NET_NS=y
-# CONFIG_CHECKPOINT_RESTORE is not set
-# CONFIG_SCHED_AUTOGROUP is not set
+CONFIG_CHECKPOINT_RESTORE=y
```

Comment:
This config has been identified as attack surface from KSPP and was intentionally disabled earlier.

Reply:
Discussed offline. May be required for boot. Will prune down in future PR.